src/rendezvous_mediator.rs

```diff
@@ -501,12 +501,13 @@ async fn direct_server(server: ServerPtr) {
                         addr,
                         err
                     );
-                    loop {
-                        if port != get_direct_port() {
-                            break;
-                        }
-                        sleep(1.).await;
-                    }
+                    // loop {
+                    //     if port != get_direct_port() {
+                    //         break;
+                    //     }
+                    //     sleep(1.).await;
+                    // }
+                    sleep(1.).await;
```

Comment:
What do you mean here? sleep a moment then retry?

My original idea is to avoid repeating retry until the user changed the port.

Reply:
the port was already allocated by another rustdesk process and could not be received. The process is about to be reallocated after that process dies.

Reply:
I think we can have clever solution, e.g.
```
ntries++
for i in 0..ntries {
checkport change
sleep
}
```
ntries is reset if port changed or no errors. max of ntries can be set to 60